chapters/pachev/main.tex

```diff
@@ -132,15 +148,22 @@ \section*{Sample Usage}
     u.x.array[:] = u.vector.array
```

Comment:
This operation cannot be used with MPI running code, as `u.vector.array` does not include ghosts. Do you want to add a `u.x.array[:local_size] = u.vector.array` here and then a scatter forward to make it parallel safe, or does your code somewhere internally check that comm.size==1?